dist/src/main/resources/wildfly/patches/standalone.xsl

```diff
@@ -972,6 +972,13 @@
 
         </managed-servers>
 
+        <platform enabled="true">
+          <file-stores   enabled="true" interval="5" timeUnits="minutes" />
+          <memory        enabled="true" interval="1" timeUnits="minutes" />
+          <processors    enabled="true" interval="1" timeUnits="minutes" />
+          <power-sources enabled="true" interval="5" timeUnits="minutes" />
```

Comment:
No, I am not sure :) We can easy change these defaults to whatever we deem the most appropriate. I just had them all enabled for now as I was testing my implementation